Automattic/course
# Address theme review feedback

We received some [feedback](https://themes.trac.wordpress.org/ticket/128905) after submitting the theme. This PR addresses that feedback:

- Removes the Theme URI.
- Adds copyright information for fonts and images.
- Removes SVG image and uses built-in search icon for search block instead.

## Testing Instructions
- Check the search page and ensure the search block displays a search icon.

![Screenshot 2022-11-15 at 8 16 20 AM](https://user-images.githubusercontent.com/1190420/201928995-c4dd9995-c80d-47f7-8c53-15434404eaac.jpg)
